katas/content/oracles/index.md

```diff
@@ -60,9 +60,9 @@ where $\ket{x_i}$ represents the state of the $i$-th qubit.
 Oracles must be unitary transformations, and follow the same rules of linear algebra as other quantum operations.
 This allows us to define quantum oracles based on their effect on the basis states - tensor products of single-qubit basis states $\ket{0}$ and $\ket{1}$.
 
-> For example, an oracle that implements a function that takes two bits of input will be defined using its effect on basis states $\ket{00}$, $\ket{01}$, $\ket{10}$, and $\ket{11}$.  
+> For example, an oracle that implements a function that takes two bits of input is defined using its effect on basis states $\ket{00}$, $\ket{01}$, $\ket{10}$, and $\ket{11}$.  
 
-There are two types of quantum oracles: phase oracles and marking oracles.  Let's take a closer look at them.
+There're two types of quantum oracles: phase oracles and marking oracles.  Let's take a closer look at them.
```

Comment:
I didn't even know this was a valid contraction. I think it reads far worse than "There are", personally.